screen_brightness_control/__main__.py

```diff
@@ -12,7 +12,9 @@ def get_monitors(args):
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(prog='screen_brightness_control')
     parser.add_argument('-d', '--display', help='the display to be used')
-    parser.add_argument('-s', '--set', type=int, help='set the brightness to this value', metavar='VALUE')
+    parser.add_argument('-s', '--set', type=str,
+                        help='set the brightness to value between 0 to 100 or in increment as string "+5" or "-5"',
```

Comment:
Just nitpicking here but I don't think the user needs to be told the bounds of the brightness percentage, but having the increment value examples is a good shout. Maybe something like:
`help='set the brightness value as a percentage or increment (eg:"+5" or "-5")'`